### ISSUE TITLE
Set headers?

### ISSUE THREAD

**Issue by retardedHacker** [chrome-php/headless-chromium-php#55]
I don't understand how I set headers. For example Accept-Language etc..

**Comment by Kozin-br** [comment]
for headers you might to use interception

**Comment by retardedHacker** [comment]
Thank you. Could you point me in the right direction? I have looked into the code but I don't see where I should "intercept".

**Comment by CyberPunkCodes** [comment]
I messed around with this for a bit, and dug through the source trying to figure it out. I wasn't able to set the headers on the initial connection, but I was able to set them on the page instance. I think setting it on the initial connection would make the most sense, and I am unsure if subsequent page requests would retain the headers or not.

```php
$page = $browser->createPage();

$page->getSession()->sendMessageSync(new Message('Network.setExtraHTTPHeaders', [
    'headers' => [
        'Accept-Encoding' => 'identity',
        'Accept-Language' => 'en-US,en;q=0.9',
    ]
]));

$page->navigate($url);
```

I haven't worked with chromium much, so maybe someone more familiar with it and this project might have a better way.
